src/TUTORIAL.md

```diff
@@ -1253,6 +1275,24 @@ module.exports = {
 };
 ```
 
+Now we support both:
+
+[JSON Patch](https://tools.ietf.org/html/rfc6902) format
+
+_Note: if using the X2 RESTful API Tester you will need to enter application/json-patch+json into the 'Content Type' form field.)_:
+
```

Comment:
I updated the API Tester so that it sets the content type automatically when PATCH method is selected, but I suppose it's still worth it mentioning it here. I'll update the API Tester screenshot in the tutorial shortly.

Reply:
If this is reliable cross-browser, and will not fail on all modern browsers, I think it's safe to remove this note. It tripped me up at the time, so I put it in, but should no longer cause anyone else problems. Let me know if you agree.

Reply:
I tested on Chrome, Safari and Firefox. Should be fine on the IE too -- the app is not doing anything crazy, pretty straightforward ReactJS/Bootstrap use-case. On the other hand, the note doesn't hurt. So... I'd say let's keep the note but say "make sure" instead of "you will need to enter".

Reply:
@levahim - done.